.env.template

```
@@ -7,3 +7,9 @@ FLASK_DEBUG=true
 
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=secret-key
+
+# Details for Trello API Credentials:
+TRELLO_API_KEY=<Required>
+TRELLO_API_TOKEN=<Required>
+TRELLO_BOARD_ID=<Required>
+TRELLO_TODO_LIST_ID=<Required>
```

Thanks for remembering to update these - it makes my time reviewing much easier!
Edit: having written that, I discovered one that had slipped the net though!
```suggestion
TRELLO_TODO_LIST_ID=<Required>
TRELLO_DONE_LIST_ID=<Required>
```